src/containers/SmartApp.jsx

```diff
@@ -28,7 +28,7 @@ class SmartApp extends FullApp {
                         this.dataAccess = new DataAccess("McodeV05SmartOnFhirDataSource");
                         this.loadPatient(smart.patient.id);
                     } else {
-                        this.dataAccess = new DataAccess(this.props.dataSource);
+                        this.dataAccess = new DataAccess(this.props.dataSource, this.props.dataSourceProps);
```

Comment:
If `dataSourceProps` is going to be a thing, should it be added to the other call in the FullApp and CoreCancerPilotApp as well? Or it's fine with me to save that until it would actually be used there.

Reply:
Yeah I thought about that too. I think it's fine to only pass it into the data sources that use it because that can be known in the factory method.